### DISCUSSION TITLE
Optimized for renewable energy.. multi objective or constraint similar to resilience?

### DISCUSSION THREAD

**Discussion by gmcclone1** [NREL/REopt-API-Analysis#162] [1 - General REopt Questions]
I found in the REopt user manual that the resilience selection maintains a least cost objective but adds a constraint which requires the "microgrid" is capable of sustaining a critical load. If I select the option for Renewable Energy does this add a maximization constraint or turn the problem into a multi objective that minimizes cost while maximizing renewable energy utilization? Thanks.

**Comment by adfarth** [comment]
If you add a min or max renewable energy input, this will add a constraint to the problem to ensure the min or max is met. 

If you choose to "include the cost of climate emissions in the objective function", then this will add another cost to the objective function, which is the lifecycle cost of energy for the site.
